pages/mesosphere/dcos/1.13/release-notes/1.13.5/index.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 1.13.5
+title: Release notes for 1.13.5
+menuWeight: 2
+excerpt: Release notes for DC/OS 1.13.5, including Open Source attribution, and version policy.
+---
+DC/OS 1.13.5 was released on 2 October 2019.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/1.13.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="http://downloads.mesosphere.com/dcos-enterprise/stable/1.13.5/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+Registered DC/OS Enterprise customers can access the DC/OS Enterprise configuration file from the [support website](https://support.mesosphere.com/s/downloads). For new customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+DC/OS 1.13.5 includes the following components:
+
+
+# Release summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+- Updated to Mesos [1.8.2-dev](https://github.com/apache/mesos/blob/adc958f553c3728aab5529de56b0ddc30c0f9b68/CHANGELOG)
+
+- Updated to Marathon 1.8.227.
+
+
+# Issues fixed in DC/OS 1.13.5
+The issues that have been fixed in DC/OS 1.13.5 are grouped by feature, functional area, or component. 
+- [enterprise]Marathon: Fixed a bug in which a service could get stuck if a failure occurred while Mesos tried to create a reservation. (MARATHON-8693) [/enterprise]
```

Comment:
This is not an enterprise-only change!

In `CHANGES.md` this is therefore listed in `dcos/dcos/blob/1.13/CHANGES.md`, and not in the dcos-enterprise `CHANGES.md`.

Reply:
Thanks for catching this! Enterprise tag deleted.